document `--project_sanity` and `--foundry` (#347)
Certora/Documentation: docs/prover/cli/options.md

**Example**
`certoraRun Bank.sol --verify Bank:Bank.spec --short_output`

(--project_sanity)=
### `--project_sanity`

**What does it do?**
Runs the builtin sanity rule on all methods in the project. If the Prover is run

> from within a git project, all .sol files in the in the git repository are added

Reviewer: Maybe?
```suggestion
from within a git project, all `.sol` files in the in the git repository are added
```

to the scene and the {ref}`builtin sanity rule <built-in-sanity>` is run on

> them. Otherwise, _all_ .sol files in the tree under the current working

Reviewer: ```suggestion
them. Otherwise, _all_ `.sol` files in the tree under the current working
```

directory are collected.

> Alternatively, a list of files could be provided and then the builtin sanity

Reviewer: ```suggestion
Alternatively, a list of files can be provided in the `.conf` file and then the builtin sanity
```
